sdk/synapse/synapse-spark/src/SparkClient.ts

```diff
@@ -0,0 +1,481 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/* eslint @typescript-eslint/member-ordering: 0 */
```

Comment:
?

Reply:
Removed

---
In reply to: [521613316](https://github.com/Azure/azure-sdk-for-js/pull/12487#discussion_r521613316) [](ancestors = 521613316)